src/utils/validation.ts

```diff
@@ -0,0 +1,15 @@
+export const reformPrice = (value: string): string =>
+  value
+    .split('')
+    .map((num, index, arr) => {
+      const needComma = (arr.length - index) % 3 === 0 && index !== 0
+
+      return needComma ? `,${num}` : num
+    })
+    .join('')
+
+export const validationNumber = (value: string, isPrice: boolean): string => {
+  const numberValue = value.replace(/[^0-9]/g, '')
+
+  return isPrice ? reformPrice(numberValue) : numberValue
+}
```

Comment:
```suggestion
export const isNumber = (value: string): boolean => {
  const numberReg = /[0-9]/g
  return numberReg.test(value)
}
```
앗 설명이 부족했군뇨..! valid내에서는 valid 역할만 하면 좋습니다!

Reply:
[요기](https://github.com/price-offer/offer-design-system/pull/35/commits/0679b5977c191cc0670edb15a87665fd7f29f4c2)에 반영 완료 ! ☺️